docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+
+To do this, open **Default apps** in **Windows Settings** and scroll down to select the option **Default Apps by Protocol**. Set the `MICROSOFT-EDGE:` protocol to Legacy Edge.
+
+### Nonconventional Browsers
+
+To use a different type of browser, use a [browser provider plugin](../extending-testcafe/browser-provider-plugin/README.md).
+
+You can search npm for plugins developed by the community. Their names begin with the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+
+You can also create your own plugin. See [Browser Provider Plugin](../extending-testcafe/browser-provider-plugin/README.md) for instructions.
+
+## Test in Headless Mode
+
+TestCafe allows you to run tests in Google Chrome and Mozilla Firefox without any visible UI shell - in the headless mode ([Chrome Headless](https://developers.google.com/web/updates/2017/04/headless-chrome), [Firefox Headless](https://developer.mozilla.org/en-US/Firefox/Headless_mode)). Use the `:headless` parameter to launch a browser in the headless mode.
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:headless')
+    .run();
+```
+
+Specify a path to the browser executable if you use a portable version of the browser. Use the [browser alias](locally-installed-browsers) instead of the `path:` prefix.
+
+```sh
+testcafe "firefox:path/to/firefox:headless" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('firefox:path/to/firefox:headless')
+    .run();
+```
+
+Note that the [Take screenshot](../guides/basic-guides/interact-with-the-page.md#take-screenshot) and [Resize window](../guides/basic-guides/interact-with-the-page.md#resize-window) features are fully supported in the headless mode.
+
+### Automation Port
+
+Chrome and Firefox require a remote control port for TestCafe to attach to the browser instance. TestCafe automatically assigns a free port but you can specify a custom port via the `cdpPort` argument for Chrome and the `marionettePort` argument for Firefox.
+
+```sh
+testcafe "chrome:headless:cdpPort=9223" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:headless:cdpPort=9223')
+    .run();
+```
+
+```sh
+testcafe "firefox:headless:marionettePort=9223" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('firefox:headless:marionettePort=9223')
+    .run();
+```
+
+## Use Chromium Device Emulation
+
+You can run test with Chromium's built-in [device emulator](https://developers.google.com/web/tools/chrome-devtools/device-mode/) in Google Chrome, Chromium and Chromium-based Microsoft Edge. To do this, use the `emulation` browser parameter.
```

Comment:
Can we say:
You can run tests with Chromium's built-in [device emulator](https://developers.google.com/web/tools/chrome-devtools/device-mode/) in Google Chrome, Chromium and Chromium-based Microsoft Edge. Use the `emulation` browser parameter.